build.xml

```diff
@@ -6,70 +6,30 @@
     <property name="path.packages" value="./packages"/>
     <property name="path.bin-console" value="./project-base/bin/console"/>
     <property name="path.framework" value="${path.packages}/framework"/>
+    <property name="path.ecs.configuration" value="${path.packages}/coding-standards/easy-coding-standard.yml"/>
 
     <import file="project-base/build-dev.xml" />
     <import file="project-base/build.xml" />
 
-    <target name="standards" depends="phplint,phpcs-fixer,phpcs,phpmd,phpstan,twig-lint,yaml-lint,eslint-check,standards-packages" description="Checks coding standards in both project-base and packages."/>
-    <target name="standards-diff" depends="phplint-diff,phpcs-fixer-diff,phpcs-diff,phpmd-diff,phpstan,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages" description="Checks coding standards on changed files in project-base and all files in packages."/>
+    <target name="standards" depends="phplint,ecs,phpstan,twig-lint,yaml-lint,eslint-check,standards-packages" description="Checks coding standards in both project-base and packages."/>
+    <target name="standards-diff" depends="phplint-diff,ecs-fix,phpstan,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages" description="Checks coding standards on changed files in project-base and all files in packages."/>
 
     <target name="standards-fix" description="Automatically fixes some coding standards violations in both project-base and packages.">
-        <phingcall target="phpcs-fixer-fix" />
-        <trycatch>
-            <try>
-                <phingcall target="phpcs" />
-            </try>
-            <catch>
-                <phingcall target="phpcbf" />
-            </catch>
-        </trycatch>
+        <phingcall target="ecs-fix" />
         <phingcall target="eslint-fix" />
         <phingcall target="standards-fix-packages" />
     </target>
 
-    <target name="standards-fix-diff" description="Automatically fixes some coding standards violations in changed files in project-base and all files in packages.">
```

Comment:
I don't see removing `standsrds-fix-diff` in PIA (yodiz), why did you remove this target? at least `eslint-fix-diff` and `standards-fix-packages` could stay here